package.json

```
@@ -10,18 +10,18 @@
   "dependencies": {
     "react": "16.4.2",
     "react-dom": "16.4.2",
-    "react-scripts": "1.1.4",
-    "react-testing-library": "5.2.1"
+    "react-scripts": "1.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "format": "prettier --write src/*"
+    "format": "prettier --write src/* src/**/*"
```

```suggestion
    "format": "prettier --write src/**/*"
```

just doing this must be enough

---

I'm on Ubuntu and having just src/**/* only found the tests directory. Wrapping it in single quotes seems to do the trick for getting everything.